components/form/Quiz.vue

```diff
@@ -336,6 +336,11 @@ export default defineComponent({
         }
         if (hasDuplicates(options.value))
           return openSnackbar("error", "Error.OptionsCannotBeSame");
+        for (let i = 0; i < options.value.length; i++) {
+          if (options.value[i].answer.length > 255) {
+            return openSnackbar("error", "Error.OptionCannotBeMoreThen256");
```

Comment:
```suggestion
            return openSnackbar("error", "Error.OptionCannotBeMoreThan256");
```

Also the translation key seems to be missing